core/src/test/scala/com/nvidia/spark/rapids/tool/qualification/QualificationSuite.scala

```diff
@@ -1376,6 +1374,50 @@ class QualificationSuite extends FunSuite with BeforeAndAfterEach with Logging {
     val logFiles = Array(s"$logDir/empty_eventlog",  s"$logDir/nested_type_eventlog")
     runQualificationTest(logFiles, "multi_run_freq_test_expectation.csv")
   }
+
+  test("test CSV qual output with escaped characters") {
+    val jobNames = List("test,name",  "\"test\"name\"")
```

Comment:
#301

Reply:
Regarding the `"",` edge case, once inside the double quotes commas need no escape and the reader will read the current value until the next `"` not used an escape character.
`"",` --> empty string value (tested)
`""",` --> unclosed value that starts with `",`
`"""",` --> single character value `"`

I'll add tests for some of the weirder potential fields that are purely double quotes and commas, without alphanumerics to delimitate the string.

Reply:
#301 -- covers a follow on issue for removing redundant calls to double quote empty string fields and return to `,`-separated lists for some of the fields in tool output, split off because some of the cleanup overlaps into the profiling tool
#330 -- P0 feature request for testing/tracking plan of spreadsheet compatibility with qualification tool output.